src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,117 @@
+@import "../utils/mixins";
+@import "../utils/vars";
+
+.page__body--with-menu {
+  overflow: hidden;
+}
+
+.menu {
+  position: fixed;
+  left: 0;
+  top: 0;
+  right: 0;
+  bottom: 0;
+
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  grid-template-rows: repeat(4, max-content);
+  z-index: 2;
```

Comment:
grid-template-rows is not needed here, as you're not using a grid layout for the .menu class. Remove this line.

Reply:
If I delete this template, layout looks very bad. I use grid.